docker/Dockerfile

```diff
@@ -1,11 +1,7 @@
-FROM rocker/r-ver:4.3.3
+FROM bioconductor/r-ver:3.18
```

Comment:
Is there a specific reason you made this change?

Reply:
Yes, and it is related to https://github.com/AlexsLemonade/scpcaTools/pull/270#discussion_r1574768161. The bioconductor images already have ~all requirements for bioconductor tools installed, so we can skip installing dependencies on our own. This image also enables binary installs of at least some Bioconductor packages, which can speed up build times quite a bit.

Reply:
Does this mean we can remove the `install_scpca_deps.sh` all together?

Reply:
> Does this mean we can remove the `install_scpca_deps.sh` all together?

It could mean that. I didn't do it quite yet because I wasn't sure 100% sure if we wanted to make the base image change.